worlds/hk/__init__.py

```diff
@@ -441,12 +443,36 @@ def set_rules(self):
             world.completion_condition[player] = lambda state: state.count("Defeated_Pantheon_5", player)
         elif goal == Goal.option_godhome_flower:
             world.completion_condition[player] = lambda state: state.count("Godhome_Flower_Quest", player)
+        elif goal == Goal.option_grub_hunt:
```

Comment:
We should populate slot data if the goal is Any as well

Reply:
if grub hunt is a valid 'any' goal, how do you want to handle the completion condition?
leave as-is but send the count in slot data so we know when to send goal complete?

Reply:
Yeah I would do it that way just so that minimal + any doesn't accidentally force something stupid (we'll leave that for minimal + random goal)

Reply:
Actually maybe a better approach for random goal would be to ensure every goal is completable in logic so that even on minimal the player would have any choice of goal